src/porepy/models/constitutive_laws.py

```diff
@@ -1765,6 +1776,34 @@ def potential_trace(
         )
         return pressure_trace
 
+    def combine_boundary_operators_tpfa_flux(
```

Comment:
I think we need access to the separate methods for Darcy and Fourier, probably passed as arguments to diffusive_flux. Let's discuss if you disagree.